test/local_integration_test.py

```diff
@@ -123,7 +123,7 @@ def check_manifest(self, response: bytes):
         """Assert that manifest contains at least one row of metadata."""
         wrapper = TextIOWrapper(BytesIO(response))
         num_rows = len(list(csv.reader(wrapper, delimiter='\t')))
-        self.assertTrue(num_rows > 1)
+        self.assertTrue(num_rows > 0)
```

Comment:
Doesn't `num_rows` include the header row? IOW, won't there always be at least one row in the TSV, even if there are no data rows?